pkgs/development/python-modules/pymyq/default.nix

```
@@ -2,25 +2,29 @@
 , aiodns
 , aiohttp
 , async-timeout
+, beautifulsoup4
 , buildPythonPackage
 , fetchFromGitHub
+, pkce
 }:
 
 buildPythonPackage rec {
   pname = "pymyq";
-  version = "2.0.14";
+  version = "3.0.0";
 
   src = fetchFromGitHub {
     owner = "arraylabs";
     repo = pname;
     rev = "v${version}";
-    sha256 = "18825b9c6qk4zcvva79hpg6098z4zqxyapnqmjsli23npw0zh67w";
+    sha256 = "sha256-uCXgJxMy5gZQBvhHnmfev/rfJXOjhLqZdOrn8SetUT0=";
   };
 
   propagatedBuildInputs = [
     aiodns
     aiohttp
     async-timeout
```

```suggestion
    aiohttp
```

According to their setup.py and requirements.txt combined. Not sure if this is complete.

---

Yes, it is complete. They others are only mentioned in the `Pipfile` which is not used.